src/api/error.js

```diff
@@ -53,6 +64,8 @@ async function handleError( e: Object, options: Object = {} ): Object {
   // matter.
   if ( e.response.status === 429 ) {
     throw new INatApiTooManyRequestsError( context );
+  } else if ( e.response.status === 401 ) {
+    throw new INatApiUnauthorizedError( context );
```

Comment:
one problem we're bumping into in 429 errors is that the context is still showing up as `null` every time. i'm not sure what the core issue is there -- maybe we're not awaiting the ` handleError` function throughout the app and we need to? I don't think this will fix the core issue or help us add context, unfortunately.